### ISSUE TITLE
BUG: Cannot run test in python

### ISSUE THREAD

**Issue by happinessee** [IronCoreLabs/ironcore-alloy#154]
Hi team,

I followed the instructions in the README to activate the virtual environment and run the example script. However, I encountered the following error:

```python
/.../workspace/ironcore-alloy/examples/python/standalone/vector-search.py:46: DeprecationWarning: There is no current event loop
  loop = asyncio.get_event_loop()
Plaintext vector:  [1.2, -1.23, 3.24, 2.37]
Traceback (most recent call last):
  File "/.../workspace/ironcore-alloy/examples/python/standalone/vector-search.py", line 47, in <module>
    loop.run_until_complete(main())
  File "/.../Library/Application Support/hatch/env/virtual/.pythons/3.12/python/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/.../workspace/ironcore-alloy/examples/python/standalone/vector-search.py", line 27, in main
    plaintext = alloy.PlaintextVector(data, "contacts", "conversation-sentiment")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: PlaintextVector.__init__() takes 1 positional argument but 4 were given
It seems that the PlaintextVector class is being called with 4 positional arguments, but it only accepts 1. This causes the example script to fail.
```

### Steps to Reproduce:
- Follow the instructions in the README to set up the environment.
- Run the example script located at examples/python/standalone/vector-search.py.

### Expected Behavior:
- The script should run without errors and demonstrate the functionality of the library.

#### Environment:
- macOS 14.6.1(23G93)
- MacBook Pro 14 (M1 Pro)
- Version of the package: 0.11.1

Would you be able to provide guidance on how to resolve this issue?

Thank you!

**Comment by skeet70** [comment]
Thanks for the report, those arguments are now named parameters instead of positional ones. [The docs](https://ironcore-alloy.readthedocs.io/en/latest/api-reference/#ironcore_alloy.ironcore_alloy.PlaintextVector) are up to date. The ones in that order are `plaintext_vector`,`secret_path`, `derivation_path`.

We'll update the example.

**Comment by skeet70** [comment]
This should be fixed by #155. You can make the changes from that PR in whichever example you're running locally, or check out the branch, if you want them working before they merge.
